src/autora/experiment_runner/experimentation_manager/example_experimentation_manager/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""
+Example experimentation manager:
+    This is an example of functions that could be included 
+    (but are not necessarily included)
+"""
+
+
+def send_conditions(args, conditions):
+    """
+    Add a description here
+    Args:
+        args: the arguments needed to set up the experiment (for example credentials)
+        conditions: the conditions to be sent
+    """
+    pass
+
+
+def check_status(args):
```

Comment:
Perhaps we could add in the description what the purpose of this function should be?